app/assets/javascripts/cascade/level/accordion.js

```diff
@@ -32,6 +53,7 @@ $(function () {
   })
   // HANDLE EXPAND TOGGLES
   $(".collapsibles-widget .toggle-expand-collapse").on('click keypress', function (event) {
+    debugger
```

Comment:
@cbryant24

Reply:
@nicholasnadel should be removed, good catch